### PR TITLE
fix Age filter bug

### PR DESCRIPTION
**Pull request by jlhogan**
## Changes

Updates age filter to display as "selected" whenever any value is selected, even if selected during guided search. To make this feel more logical/reasonable, also updates age filter to be clear-able in the filter dropdown
## Checklist

- ~[ ] if adds a new page, adds accessibility tests~
- ~[ ] if adds a new page or new interaction, adds e2e test~
- ~[ ] if adds a new page or new interaction, sends google analytics events and updates [GA doc]~(https://docs.google.com/document/d/1DfQDUNZPO3B352EhUwXp0el6qrAWz8Jq9cTa1yv8Ty4/edit)
- ~[ ] if changes filter functionality, updates [filters doc](https://docs.google.com/document/d/1yEdo7IpHCtEKNNF6FMLapBUgcPw_8CY7wGwxKTdtJrU/edit)~

Fixes https://app.asana.com/0/1202690670605811/1202973748936557/f

### CODE REVIEW

File: src/components/Search/Filters/Control/DesktopControl.tsx [hunk 1]
```diff
@@ -111,16 +113,18 @@ function DesktopControl({
         </DesktopControlDropdown>
         <DesktopControlDropdown
           title={t("ageTitle")}
-          hasSelection={!!filters.age && showAgeActive}
+          hasSelection={!!filters.age}
+          clear={() => {
+            const updatedFilters = { ...filters };
```
Comment (cathydeng) [inline src/components/Search/Filters/Control/DesktopControl.tsx:118]: nit: you can also do

```
const {age, ...updatedFilters} = filters
setFilters(updatedFilters)
```

File: src/components/Search/Filters/Control/DesktopControl.tsx [hunk 2]
```diff
@@ -11,6 +11,7 @@ import AccessibilityInput from "../AccessibilityInput";
 import DistanceInput from "../DistanceInput";
 import ControlToggles from "./DesktopControlToggles";
 import AgeGroupInput from "../AgeGroupInput";
+import { useLocation, useNavigate } from "react-router-dom";
```
Comment (cathydeng) [inline src/components/Search/Filters/Control/DesktopControl.tsx:14]: seems like useEffect, useLocation, and useNavigate aren't used?

**Review by cathydeng — APPROVED** [review]
just lil things, functionality makes sense to me!

File: src/components/Search/Filters/Control/DesktopControl.tsx (hunk 2 above)
Comment (jlhogan) [reply src/components/Search/Filters/Control/DesktopControl.tsx:14]: yup this was from a much more involved and very unnecessary first attempt at a fix

**Pull request merged**